### PR TITLE
Edits on Import from ODK dialog

### PR DESCRIPTION
**Pull request by stevenndungu**

### CODE REVIEW

**Comment by stevenndungu** [comment]
Fixes #1728

File: instat/dlgImportFromODK.vb [hunk 1]
```diff
@@ -88,7 +88,7 @@ Public Class dlgImportFromODK
         TestOKEnabled()
     End Sub
 
-    Private Sub ucrInputPassword_NameChanged() Handles ucrInputPassword.NameChanged
+    Private Sub ucrInputPassword_NameChanged()
```
Comment (lilyclements) [inline instat/dlgImportFromODK.vb]: I think we would still want the "Handles ucrInputPassword.NameChanged" here?

Comment (stevenndungu) [inline instat/dlgImportFromODK.vb]: You are right. It got deleted when I replaced the input box that was existing before (by deleting). Thanks

File: instat/dlgImportFromODK.Designer.vb [hunk 2]
```diff
@@ -146,15 +136,23 @@ Partial Class dlgImportFromODK
         Me.chkViewPassword.Text = "View Password"
         Me.chkViewPassword.UseVisualStyleBackColor = True
         '
+        'ucrInputPassword
+        '
+        Me.ucrInputPassword.IsReadOnly = False
+        Me.ucrInputPassword.Location = New System.Drawing.Point(92, 63)
+        Me.ucrInputPassword.Name = "ucrInputPassword"
+        Me.ucrInputPassword.Size = New System.Drawing.Size(137, 21)
```
Comment (lilyclements) [inline instat/dlgImportFromODK.Designer.vb]: In the issue, Danny said "text box for that is sort of cut off on the dialog". By this I assume he means like this?

image

In which case we want the size to be a little longer, perhaps more like (137,30) ?

File: instat/dlgImportFromODK.vb [hunk 3]
```diff
@@ -138,7 +138,7 @@ Public Class dlgImportFromODK
         End If
     End Sub
 
-    Private Sub UsernamePassword_ContentsChanged() Handles ucrInputUsername.ContentsChanged, ucrInputPassword.ContentsChanged
+    Private Sub UsernamePassword_ContentsChanged() Handles ucrInputUsername.ContentsChanged
```
Comment (lilyclements) [inline instat/dlgImportFromODK.vb]: Additionally here I think we still want to have "ucrInputPassword.ContentsChanged" after ucrInputUsername.ContentsChanged?

**Comment by stevenndungu** [comment]
Yes, I corrected that. By choosing to delete the previous textbox that
seemed cut then replacing with another one. Check how it looks now.

**Comment by lilyclements** [comment]
@dannyparsons this is ready for you to review.

**Comment by dannyparsons** [comment]
Thanks @lilyclements 
@stevenndungu does this also address the font size issue I mentioned?

**Comment by stevenndungu** [comment]
Yes it does. It now has normal font type and size

**Pull request merged**